opennms-config-tester/src/test/java/org/opennms/netmgt/config/tester/ConfigTesterTest.java

```diff
@@ -672,6 +672,67 @@ public void testOpennmsPropertiesD() throws IOException {
         testConfigFile("opennms.properties.d");
     }
 
+    @Test
+    public void testValidImports() throws IOException {
+        setUpOpennmsHomeInTmpDir();
+        final String validXml = "<model-import xmlns=\"http://xmlns.opennms.org/xsd/config/model-import\" date-stamp=\"2019-04-04T07:49:24.053+02:00\" foreign-source=\"ABC\" last-import=\"2019-04-04T07:49:30.777+02:00\"/>";
+
+        final Path etcDir = java.nio.file.Files.createDirectories(Paths.get(this.opennmsHomeDir.getPath(), "etc/imports"));
+        final File file = new File(etcDir.toFile(), "ABC.xml");
+        Files.write(validXml, file, StandardCharsets.UTF_8);
```

Comment:
Each test seem to do the same thing, and besides the `valid/invalidXml` nothing changes.
Can we refactor this to not use as much duplicated code?

Reply:
done